tests/migration/reset_scc_addons_values.pm

```diff
@@ -0,0 +1,64 @@
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Swap the values of SCC_ADDONS before migration.
+# In MU jobs, the SCC_ADDONS is used to add the products' MU repos;
+# In migration jobs, the setting is also used to register modules.
+# On SLES12SPx and SLES15SPx, the modules name are different, so need
+# to swap SCC_ADDONS values to sles15SP5 modules nemes before migration.
```

Comment:
It's better to add description that this test re-create incidents repos base on the swapped SCC_ADDONS.

Reply:
Updated, thanks for review.